faces/4.0/renderkitdoc/HTML_BASIC/renderkit-summary.html

```diff
@@ -454,6 +454,6 @@ <H2>HTML_BASIC RenderKit </H2><BR>
 
 <HR>
 
-Copyright � 2002-2010 Oracle America, Inc. All Rights Reserved.
+Copyright &#169; 2002-2010 Oracle America, Inc. All Rights Reserved.
```

Comment:
Are you sure about this copyright? I thought it needs to be under EF license.

Reply:
Not sure no, I was wondering about it too as I updated it. It was inherited like that. I'll change it to the regular Eclipse one.